pkgs/by-name/pi/pixi/package.nix

```diff
@@ -72,7 +72,10 @@ rustPlatform.buildRustPackage rec {
     "--skip=test_purl_are_added_for_pypi"
 
     # `/usr/bin/env` required
```

Comment:
maybe outside of the scope for this PR, but `/bin/sh` and `/usr/bin/env` are the two paths guaranteed in nix/nixos. Can you expand on this command?

Reply:
Not available during build. 

> Can you expand on this comment?

Sure thing.